sources/packages/api/src/route-controllers/institution/institution.institutions.controller.ts

```diff
@@ -218,7 +219,7 @@ export class InstitutionInstitutionsController extends BaseController {
     }
     // Check if the user being added to th institution belongs to the institution.
```

Comment:
There is a E missed in the end "Check if the user being added to th" comment. Please change. I am aware its not part of your PR. Thanks